com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/IDebugAdapterContext.java

```diff
@@ -59,6 +72,10 @@
 
     void setClientPathsAreUri(boolean clientPathsAreUri);
 
+    void setSupportsRunInTerminalRequest(boolean supportsRunInTerminalRequest);
+
+    boolean isSupportsRunInTerminalRequest();
```

Comment:
**is** is not necessary for isSupportsRunInTerminalRequest

Reply:
nice.